src/main/java/net/sf/jabref/JabRef.java

```diff
@@ -898,11 +889,11 @@ private static ParserResult importFile(String argument) {
      * @param argument See importFile.
      * @return ParserResult with setToOpenTab(true)
      */
-    private static ParserResult importToOpenBase(String argument) {
-        ParserResult result = JabRef.importFile(argument);
+    private static Optional<ParserResult> importToOpenBase(String argument) {
+        Optional<ParserResult> result = JabRef.importFile(argument);
 
-        if (result != null) {
-            result.setToOpenTab(true);
+        if (result.isPresent()) {
+            result.get().setToOpenTab(true);
         }
```

Comment:
in that case, use the following:

```
optional.ifPresent(value -> value.setToOpenTab(true));
```

The function is only executed if the optional is present.